.cargo/config.toml

```
@@ -4,4 +4,4 @@ xfast = "run --package xtask --"
 xtask = "run --package xtask --release --"
 
 [build]
-rustflags = ["-Dclippy::all", "-Dwarnings"]
+# rustflags = ["-Dclippy::all", "-Dwarnings"]
```

For ease of development, will go away in the final version.

```suggestion
rustflags = ["-Dclippy::all", "-Dwarnings"]
```